docs/Options.md

```diff
@@ -429,6 +429,7 @@ Show activated conda virtual environment. Disable native conda prompt by `conda
 | `SPACESHIP_CONDA_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after the conda virtualenv section |
 | `SPACESHIP_CONDA_SYMBOL` | `🅒·` | Character to be shown before conda virtualenv section |
 | `SPACESHIP_CONDA_COLOR` | `blue` | Color of conda virtualenv environment section |
+| `SPACESHIP_CONDA_VERBOSE` | `true` | Toggle to truncate environment names under custom prefix |
```

Comment:
Why do we need this?

Reply:
Default behavior of `conda` is to return name/prefix depending on their logic, So many users **would expect to see complete path** with custom prefixes. So adding an option keeps the `conda` behavior unchanged with _possibility_ to show only the basename of path. 

> `conda` has _very specific_ naming convention. `CONDA_DEFAULT_ENV` is truncated only if their immediate parent name is `envs`. In case of anything else, they just return the `prefix`. Makes sense in someway, Because **sometimes user may have test environment with same name but with different prefix.** 

https://github.com/denysdovhan/spaceship-prompt/issues/589#issuecomment-448868329